frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/fossologyClearing.jspf

```diff
@@ -76,16 +80,24 @@
                         </div>
                     </div>
                     <div class="row mt-4">
-                        <div class="col">
-                            <div class="auto-refresh text-right">
+                        <div class="col-md-12">
+                            <div class="auto-refresh text-left">
```

Comment:
Please remove `text-left` class if not required

Reply:
Done